index.html

```diff
@@ -5,18 +5,18 @@
     <meta charset="utf-8" />
   </head>
   <body>
+    <center>
```

Comment:
this tag is not supported by HTML5 https://www.w3schools.com/tags/tag_center.asp